src/lib/OpenEXR/ImfZstdCompressor.cpp

```diff
@@ -0,0 +1,110 @@
+#include <cstring>
+#include "ImfZstdCompressor.h"
+
+#include "blosc2.h"
+#include "IlmThreadPool.h"
+#include "ImfChannelList.h"
+#include "ImfMisc.h"
+namespace
+{
+class BloscInit
+{
+public:
+    static void Init () { getInstance (); }
+    BloscInit (const BloscInit&) = delete;
+    BloscInit& operator= (const BloscInit&) = delete;
```

Comment:
formatting diff on my end:
```suggestion
    BloscInit (const BloscInit&)            = delete;
    BloscInit& operator= (const BloscInit&) = delete;
```